### PR TITLE
Address Invalid scenario for nfs export settings

### PR DESCRIPTION
**Pull request by NavisJ**
# Description
Modify import function to include zone name
Add maximum validator for max_file_size
Parse default snapshot value to be acceptable for update API

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# ISSUE TYPE

- Bugfix Pull Request

##### RESOURCE OR DATASOURCE NAME
NFS Export Settings

##### OUTPUT

```paste below

```
##### ADDITIONAL INFORMATION

```paste below

```
# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [x] I have maintained at least 80% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have maintained backward compatibility

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [x] Unit tests
- [x] Acceptance tests

### CODE REVIEW

**Comment by NavisJ** [comment]
run e2e test

File: powerscale/provider/nfs_export_settings_resource.go [hunk 1]
```diff
@@ -664,6 +665,9 @@ Note that, NFS Export Settings is the native functionality of PowerScale. When c
 				Optional:            true,
 				Description:         "Specifies the maximum file size for any file accessed from the export. This parameter does not affect server behavior, but is included to accommodate legacy client requirements.",
 				MarkdownDescription: "Specifies the maximum file size for any file accessed from the export. This parameter does not affect server behavior, but is included to accommodate legacy client requirements.",
+				Validators: []validator.Int64{
+					int64validator.AtMost(9223372036854775807),
```
Comment (taohe1012) [inline powerscale/provider/nfs_export_settings_resource.go:669]: the best practice when do release, we cannot use such big num const here

Comment (NavisJ) [reply powerscale/provider/nfs_export_settings_resource.go:669]: ok. Thank you

**Comment by NavisJ** [comment]
run e2e test

**Review by taohe1012 — APPROVED** [review]

File: powerscale/provider/nfs_export_settings_resource_test.go [hunk 2]
```diff
@@ -222,6 +238,12 @@ resource "powerscale_nfs_export_settings" "test" {
 }
 `
 
+var nfsExportSettingsSnapshotResourceConfig = `
+resource "powerscale_nfs_export_settings" "test" {
+	snapshot = "-"
```
Comment (baoy1) [inline powerscale/provider/nfs_export_settings_resource_test.go:243]: Should we mention `-` meaning `default` in doc?

Comment (NavisJ) [reply powerscale/provider/nfs_export_settings_resource_test.go:243]: Good point! 
Modified.

**Comment by NavisJ** [comment]
run e2e test

**Review by baoy1 — APPROVED** [review]

**Pull request merged**